devicetypes/smartthings/zooz-multisiren.src/zooz-multisiren.groovy

```diff
@@ -0,0 +1,198 @@
+/**
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ *	Aeon Siren
+ *
+ *	Author: SmartThings
+ *	Date: 2019-03-18
+ */
+ 
+metadata {
+ definition (name: "Zooz Multisiren", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.siren", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, vid: "generic-siren-11") {
```

Comment:
the `runLocally`, `executeCommandsLocally`, and `minHubCoreVersion` shouldn't be necessary.

Reply:
ok